### DISCUSSION TITLE
Not able to add custom Viz Hello World to Apache Superset

### DISCUSSION THREAD

**Discussion by GaryMam** [apache/superset#15932] [Q&A / Help]
Hi All,
I am trying to create Hello World custom viz in apache superset and able to create plugins but getting error while restarting **dev server.** Do you know how to fix it?Also attaching error

Steps I followed :
https://superset.apache.org/docs/installation/building-custom-viz-plugins

Superset I am using : http://localhost:8080/superset/welcome/

Error in the below steps. Also attaching error :

Restart your webpack dev server with npm run dev-server. You'll know it worked if you see a line stating [Superset Plugin] Use symlink source for @superset-ui/plugin-chart-hello-world @ ^0.0.0.

[Error Dev Sever.txt](https://github.com/apache/superset/files/6895389/Error.Dev.Sever.txt)

**Comment by rusackas** [comment]
It looks like it's having a hard time with the Theme, which is part of Superset-UI. Did you install all the packages in both repos?

1) In `superset-ui` (the root directory) you need to do a `yarn install` to make sure all the plugins (including your new one) get their dependencies installed
2) In `superset` you need to `npm install` _before_ doing the `npm link` step. Note that if you do the `npm-install` _after_ the `npm link` step, it overwrites/breaks the link.

Note that your webpack logs you attached DID show `[Superset Plugin] Use symlink source for @superset-ui/plugin-chart-hello-world @ ^0.0.0` which means your `npm link` step DID work.

Hope that helps.

**Reply by dandandan1983** [reply]
@rusackas maybe you will can help me?  I'm using Apache Superset 1.3.2 (not in docker), ubuntu 19.

I created (yo @superset-ui/superset) and build (yarn build) custom plugin. It works fine then i run "npm run dev-server". When I run "yarn build" plugin. I think that i will see this custom chart when i run "superset run -p 8080". I didn't see my chart. What i need to change, if i want to see my custom chart? Thanks
